source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1308,10 +1308,11 @@ Error handling
 Network error during server check
 `````````````````````````````````
 
-When a server `check`_ fails due to a network error,
-the client SHOULD clear its connection pool for the server:
+When a server `check`_ fails due to a network error (including a timeout),
```

Comment:
The spec defines terms "network error" and "network timeout", should that second term be used here?

Reply:
Changed.